cas-server-core/src/main/java/org/jasig/cas/CentralAuthenticationServiceImpl.java

```diff
@@ -279,14 +279,14 @@ public ServiceTicket grantServiceTicket(
         getAuthenticationSatisfiedByPolicy(ticketGrantingTicket, new ServiceContext(service, registeredService));
 
         final String uniqueTicketIdGenKey = service.getClass().getName();
-        if (!this.uniqueTicketIdGeneratorsForService.containsKey(uniqueTicketIdGenKey)) {
-            logger.warn("Cannot create service ticket because the key [{}] for service [{}] is not linked to a ticket id generator",
-                    uniqueTicketIdGenKey, service.getId());
-            throw new UnauthorizedSsoServiceException();
-        }
-        
-        final UniqueTicketIdGenerator serviceTicketUniqueTicketIdGenerator =
+        logger.debug("Looking up service ticket id generator for [{}]", uniqueTicketIdGenKey);

```

Comment:
this scenario can pretty much only happen if someone doesn't test their services (i.e. they configured SAML, but never tested SAML integration)  Its unclear why we would want to hide that fact.